docs/dev/perl5/index.html

```diff
@@ -27,7 +27,7 @@ <h3>Releases</h3>
     <ul>
       <li>
         Latest development release
-        is <a href="https://perldoc.perl.org/5.41.1/perldelta">Perl 5.41.1</a>
+        is <a href="https://perldoc.perl.org/5.41.1/perldelta">Perl 5.41.2</a>
```

Comment:
I think you also want to modify the URL itself.

Reply:
ah nuts, this is what happens when I send a PR when tired...